index.html

```diff
@@ -145,13 +145,13 @@ <h3>CropTarget Motivation</h3>
       <section id="crop-target">
         <h3><dfn>CropTarget</dfn> Definition</h3>
         <p>
-          CropTarget is an intentionally empty, opaque identifier that exposes nothing. Its sole
-          purpose is to be handed to {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
+          CropTarget is an intentionally empty, opaque identifier. Its purpose is to be handed to
+          {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
         </p>
         <pre class="idl">
           [Exposed=(Window,Worker), Serializable]
           interface CropTarget {
-            // Intentionally empty; just an opaque identifier.
+            [SecureContext] static Promise&lt;CropTarget&gt; fromElement(Element element);
```

Comment:
We may want to add a note explaining why `[SecureContext]` is not on the interface.